src/sourmash_plugin_test1.py

```diff
@@ -58,19 +63,19 @@ def add(self, ss):
 #
 
 
-class ScriptsCommand_XYZ:
-    command = "xyz"
-    description = "do something xyz"
+class ScriptsCommand_Test1:
```

Comment:
This adds a command-line command `sourmash scripts test1 <filename_test> -x/--xyz`.